src/linker/Linker.Dataflow/LocalDataFlowAnalysis.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using ILLink.Shared.DataFlow;
+using ILLink.Shared.TrimAnalysis;
+using Mono.Cecil;
+using Mono.Linker.Steps;
+using MultiValue = ILLink.Shared.DataFlow.ValueSet<ILLink.Shared.DataFlow.SingleValue>;
+
+namespace Mono.Linker.Dataflow
+{
+	abstract class LocalDataFlowAnalysis<TTransfer>
```

Comment:
I think this is unnecessary abstraction - at least for now. I would simply hardcode the ReflectionScanner into this.

Reply:
Removed the abstract class